src/conversions.jl

```diff
@@ -751,6 +789,8 @@ function pysequence_query(o::PyObject)
         return typetuple(pytype_query(PyObject(ccall((@pysym :PySequence_GetItem), PyPtr, (PyPtr,Int), o,i-1)), PyAny) for i = 1:len)
     elseif pyisinstance(o, pyxrange[])
         return AbstractRange
+    elseif pyisinstance(o, pyslice[])
+        return PySlice
```

Comment:
We don't default to creating no-copy wrappers of other Python sequences (e.g. lists, numpy arrays), no probably slices should be no different.

Reply:
I removed these two lines.

Reply:
I have now removed all the pyslice code from this pull request. It is not needed here.